### DISCUSSION TITLE
CPL for JLCPCB MailBox_Guard_Boom

### DISCUSSION THREAD

**Discussion by Vad1mka2022** [PricelessToolkit/MailBoxGuard#45] [Ideas]
Hello.
Maybe someone has already ordered the boards and can share the file CPL ? Sample:
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/100032847/AppData/Local/Packages/oice_16_974fa576_32c1d314_2feb/AC/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/100032847/AppData/Local/Packages/oice_16_974fa576_32c1d314_2feb/AC/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link=blue vlink=purple>

Designator | Mid X | Mid Y | Layer | Rotation
-- | -- | -- | -- | --
C1 | 95.0518mm | 22.6822mm | Top | 270

</body>

</html>

**Comment by PricelessToolkit** [comment]
Hi, I intentionally did not upload this specific file. I would like to clarify that the project files provided are specifically intended for the purpose of ordering Printed Circuit Boards (PCBs) and for manual assembly. They are not meant to facilitate assembly services in China. If you are in search of assembled and tested boards, I wish to remind you that these are available for direct purchase from my shop. Could you please inform me about the total cost of assembly and delivery that you have decided to proceed with, using your selected provider?

**Comment by Vad1mka2022** [comment]
Hello.

 I can't find out how much it would cost to build a board without this
file. 5 pcs of boards without components is $4.5 including delivery. I can
solder everything myself, but in my region there weren’t two components and
I thought of ordering ready-made ones if it’s not expensive, but if the
price is high enough I’ll look for something to replace the components
with. I understand that this is an original project and am grateful for it,
but I’m not sure whether it can work in my case, 5 floors of concrete walls
with reinforcement, and I’m not ready to spend a lot of money to try and
find out that it doesn’t work. If you do not distribute copyright files, I
understand your decision.

 Thanks again for the project.

śr., 21 lut 2024, 16:38 użytkownik PricelessToolkit <
***@***.***> napisał:

**Reply by PricelessToolkit** [reply]
Components you can buy from lcsc.com and you will get a shipping discount if you have a PCB order on JLCPCB.

PS: Only the Lora 868 module price is 8 euros.

**Comment by PricelessToolkit** [comment]
The decision not to share CPL file is not meant to hinder community members from placing assembly orders. Instead, it aims to prevent the project from appearing on platforms such as AliExpress without authorization.

**Comment by Vad1mka2022** [comment]
Thanks for the tip. I didn't know you could order components when ordering
the board. I managed to buy the Laura module for $4, the rest of the
components combined cost less than $1. prices depend greatly on the region,
as do salaries. Total total budget for one board: 1$ PCB, 4$ Lora, 1$
components = 6$.

śr., 21 lut 2024, 16:51 użytkownik PricelessToolkit <
***@***.***> napisał:
